File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraRichEdit.Export;
using DevExpress.XtraRichEdit.Import;

namespace Eml {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();

            EmlDocumentExporter.Register(richEdit);
            EmlDocumentImporter.Register(richEdit);

            richEdit.LoadDocument("Hey, look at this!.eml");
        }

    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraRichEdit.Export
Imports DevExpress.XtraRichEdit.Import

Namespace Eml
	Public Partial Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()

			EmlDocumentExporter.Register(richEdit)
			EmlDocumentImporter.Register(richEdit)

			richEdit.LoadDocument("Hey, look at this!.eml")
		End Sub

	End Class
End Namespace